extensions/interactions/GraphInput/GraphInputRulesServiceSpec.js

```diff
@@ -180,58 +180,58 @@ describe('Graph Input service', function() {
     });
   });
 
-  describe('\'is isomorphic to\' rule', function() {
-    var emptyGraph = function() {
-      return {
-        vertices: [],
-        edges: [],
-        isDirected: false,
-        isWeighted: false,
-        isLabeled: false
-      };
+  var emptyGraph = function() {
+    return {
+      vertices: [],
+      edges: [],
+      isDirected: false,
+      isWeighted: false,
+      isLabeled: false
     };
+  };
 
-    var nullGraph = function(numVertices) {
-      var graph = emptyGraph();
-      for (var i = 0; i < numVertices; i++) {
-        graph.vertices.push({
-          label: '',
-          x: 0.0,
-          y: 0.0
-        });
-      }
+  var nullGraph = function(numVertices) {
+    var graph = emptyGraph();
+    for (var i = 0; i < numVertices; i++) {
+      graph.vertices.push({
+        label: '',
+        x: 0.0,
+        y: 0.0
+      });
+    }
+    return graph;
+  };
+
+  var cycleGraph = function(numVertices) {
```

Comment:
I think these graphs probably ought to all have 'undirected' in their names. It would be nice if you could find some standard directed classes of examples.